lib/chef/knife/user_create.rb

```diff
@@ -27,7 +26,7 @@ class UserCreate < Knife
       attr_accessor :user_field
 
       deps do
-        require_relative "../user_v1"
```

Comment:
We can safely assume v1 support as the minimum API version at this point so nothing here should be moving back to the "user" class. Everything should just code to the _v1 APIs.